Sources/EventProducer/Monitoring/MonitoringQueue.swift

```diff
@@ -11,7 +11,7 @@ public final class MonitoringQueue {
 	}
 
 	private func setup() throws {
-		guard let databaseURL = FileManagerHelper.shared.eventQueueDatabaseURL else {
+		guard let databaseURL = FileManagerHelper.shared.monitoringQueueDatabaseURL else {
```

Comment:
@emin-grbo I would test this also and see. 
1) try reverting the changes, use @asendra fix and see what happens 
2) try with both changes too